data/mods/gen9ssb/moves.ts

```diff
@@ -626,7 +626,60 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		},
 		target: "normal",
 		type: "Flying",
+	},
 
+	// Hydrostatics
+	hydrostatics: {
+		accuracy: 100,
+		basePower: 90,
+		category: "Special",
+		name: "Hydrostatics",
+		shortDesc: "70% +1 SpA, 50% prz, +water effectiveness. Tera-ed: Water-type, 80% +1 SpA, 60% to Soak, super-effective against Water",
```

Comment:
this still doesn't even come close to fitting in the description box, but you did your best